### ISSUE TITLE
More detailed Linux install instructions

### ISSUE THREAD

**Issue by helgridly** [jkbonfield/io_lib#5]
As a newbie I had a bit of a fun time getting this installed on Ubuntu 14.04 this morning. Notes below if you'd like to integrate them with your README:

```
git clone git@github.com:jkbonfield/io_lib.git
sudo apt-get install autoconf build-essential automake autogen libtool shtool autotools-dev zlib1g-dev
autoreconf -i
./configure
make
make install
```

**Comment by jkbonfield** [comment]
These tools are only necessary if building from git clone, not from the release tarballs which already have the configure script created for you.

However I'll add a section about this in the README file.

**Comment by helgridly** [comment]
Ah, I see - I flipped through the readme and since all I saw was building instructions I assumed that was how to install. Didn't think to check releases, my bad!

**Comment by jkbonfield** [comment]
Resolved by d4db6015ba8b6ec78e95895242b2d34b15dfa83b
